File: Kotlin/프로그래머스/1/12901. 2016년/2016년.kt
class Solution {
    fun solution(a: Int, b: Int): String {
        var answer = ""


        val days = arrayOf(31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31)
        val dates = arrayOf("THU","FRI","SAT","SUN","MON","TUE","WED")

        var totalDays = 0
        for (i in 0 until a - 1) {
            totalDays += days[i]
        }
        totalDays += b

        return dates[totalDays % 7]
    }
}

File: C#/프로그래머스/1/12901. 2016년/2016년.cs
public class Solution {
    public string solution(int a, int b) {
        string answer = "";
        int[] days = new int[] {31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};
        string[] date = new string[] {"FRI","SAT","SUN","MON","TUE","WED","THU"};
        
        int totalDays = b;
        
        for(int i = 0; i < a - 1; i++)
        {
            totalDays += days[i];
        }
        
        
        return date[totalDays % 7 - 1 < 0 ? 6 : totalDays % 7 - 1];
    }
}